manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "affirm-api",
   "vendor": "logixalpartnerus",
-  "version": "1.3.14",
+  "version": "1.3.15",
```

Comment:
Please revert these changes to `manifest.json`. The vendor should be `vtex` and version should be `1.3.6`